website/docs/src/content/component-reference/storage-components/search-reference.mdx

```diff
@@ -0,0 +1,388 @@
+---
+title: Full-text & vector search reference
+description: Vector search components for semantic search and RAG workflows.
+---
+
+# Search Reference
+
+API reference for GenSX Cloud search components.
```

Comment:
I wonder if we should include a link to the turbopuffer docs and say that the query/upsert etc closely match the turbopuffer js sdk. Otherwise some advanced query params will be hard to understand.